src/python/grpcio/grpc/_adapter/_low.py

```diff
@@ -195,26 +195,30 @@ def start_batch(self, ops, tag):
         translated_op = cygrpc.operation_send_initial_metadata(
             cygrpc.Metadata(
                 cygrpc.Metadatum(key, value)
-                for key, value in op.initial_metadata))
+                for key, value in op.initial_metadata),
+            op.write_flags)
```

Comment:
Yes - I suppose I should have said "some functions that have nothing to do with writing and some that do".